core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -327,6 +327,25 @@ public T keepAliveWithoutCalls(boolean enable) {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Sets the retry buffer size in bytes. If the buffer limit is exceeded, no RPC
+   * could retry at the moment, and in hedging case all hedges but one of the same RPC will cancel.
+   * The implementation may only estimate the buffer size being used rather than count the
+   * exact physical memory allocated. The method does not have any effect if retry is disabled.
```

Comment:
How can retry be disabled?

Reply:
How would a client go about disabling retry? For example: what API would they use?

Reply:
I think that would be `ManagedChannelBuilder.disableRetry()`
https://github.com/grpc/proposal/blob/master/A6-client-retries.md#disabling-retries